packages/CalendarCarousel/index.tsx

```diff
@@ -266,7 +348,7 @@ function CalendarCarousel<T>({
     const calendarDays = [...prevDays, ...days, ...nextDays];
```

Comment:
Try to add `Date` params only here. Adding whole views and passing this down looks bit awkward.